### DISCUSSION TITLE
[Google Search]For Chinese user, redirect google.cn to google.com.hk

### DISCUSSION THREAD

**Discussion by BRabb1t** [mshibanami/redirect-web#38] [Redirect Rules]
大陆苹果用户 优雅地在safari上使用google搜索 添加一下规则 无需修改地区
Redirect from: ^https://m\.so\.com/s\?q=([^&]*).* (Mode: regular expression)
Redirect to: https://www.google.com.hk/search?q=$1
Application: Default (Safari)
Capture Group Processing: <empty>
Excluded URL Patterns: <empty>
Examples: https://m.so.com/s?q=nihao -> https://www.google.com.hk/search?q=nihao

**Comment by mshibanami** [comment]
@BRabb1t 很好！标题说它是从google.cn → google.com.hk，但实际上是从m.so.com → google.com.hk吗？

**Reply by BRabb1t** [reply]
when Chinese user set google as default search engine, safari will use google.cn, 
using this redirect-web extension, user set 360 search as default search engine and add this rule, it will use google.com.hk
thank you for developing this extension👍

**Reply by mshibanami** [reply]
@BRabb1t Ah, that's interesting. Thanks for clarifying! That makes perfect sense, and your rule must be great for users in China! Do you mind if I add your rule to the library section in the app, where users can get useful predefined rules? 😄

**Reply by BRabb1t** [reply]
sure👍

**Reply by mshibanami** [reply]
(I'll definitely add this rule to the library, but I'll make some backend changes first so this rule will be displayed to those who are in China only.)
